bin/marked

```diff
@@ -5,18 +5,18 @@
  * Copyright (c) 2011-2013, Christopher Jeffrey (MIT License)
  */
 
-var fs = require('fs'),
-    path = require('path'),
-    marked = require('../');
+const fs = require('fs'),
```

Comment:
Are we dropping support for old versions of Node? I think const is Node 4+?

Reply:
I don't see any reason to support obsolete versions of node. If someone is not updating their version of node they probably aren't updating their version of marked either.

We still support es5 browsers with `lib/marked.js` though